project.md

```diff
@@ -0,0 +1,96 @@
+# Rock-Paper-Scissors on the Micro:bit: A Semester Capstone Project
+
+## Abstract
+
+The goal of this semester capstone project is to demonstrate how the micro:bit—a pocket-sized computer designed for education—can be utilized to create an interactive and multiplayer game of Rock-Paper-Scissors. This paper highlights the technologies, inputs, outputs, and communication techniques used, to serve as a guide and inspiration for high school students embarking on their own semester projects.
+
+## Table of Contents
+
+1. Introduction
+2. Requirements and Pre-requisites
+3. Inputs and Outputs
+4. Communication Techniques
+5. Code Architecture
+6. Challenges and Lessons Learned
+7. Conclusion
+
+---
+
+## 1. Introduction
+
+The micro:bit is a versatile tool capable of a multitude of functions, ranging from simple displays to complex radio communications. We'll be taking advantage of its features to create a Rock-Paper-Scissors game that can be played in two modes—single-player against a computer and multiplayer via radio communication.
+
+---
+
+## 2. Requirements and Pre-requisites
+
+**Hardware Required:**
+- micro:bit (2 units for multiplayer)
+- USB cables for programming and power
+
+**Software Required:**
+- Python editor compatible with micro:bit (e.g., Mu)
+
+**Skills Required:**
+- Basic understanding of Python programming
+- Basic understanding of micro:bit's features
+
+---
+
+## 3. Inputs and Outputs
+
+### Inputs
+
+1. **Button A**: Cycles through the options (Rock, Paper, Scissors)
+2. **Button B**: Confirms the selection
+3. **Both Buttons**: Switches to multiplayer mode and selects host/client roles
+
+### Outputs
+
+1. **LED Display**: 
+    - Shows various emojis to indicate winner, loser, or tie
+    - Displays the current selection (Rock, Paper, Scissors)
+
+---
+
+## 4. Communication Techniques
+
+### Radio Communication
+
+The micro:bit has built-in radio communication capabilities which allow it to send and receive messages. In our project, this is used to:
+
+1. **Announce Host and Client Status**: When a micro:bit is set to host, it sends a 'HOST' message. The client listens for this message to establish a connection.
+2. **Synchronize Choices**: Once connected, each micro:bit sends its choice to the other to determine the winner.
+
+---
+
+## 5. Code Architecture
+
+The code is divided into various functional blocks:
+
+1. **Initialization**: Setting up variables and default modes.
+2. **Display Logic**: Code to handle what is shown on the LED screen.
+3. **Single-player Logic**: Handling interactions for single-player mode.
+4. **Multiplayer Logic**: Handling radio communications and interactions for multiplayer mode.
```

Comment:
This is a comment :)

# heading
## sub heading

List 
 1. thing
 2. other thing